public/app/features/explore/ExplorePaneContainer.tsx

```diff
@@ -22,6 +23,19 @@ import Explore from './Explore';
 import { initializeExplore, refreshExplore } from './state/explorePane';
 import { lastSavedUrl, cleanupPaneAction } from './state/main';
 
+const getStyles = () => {
+  return {
+    explore: css`
+      display: flex;
+      flex: 1 1 auto;
+      flex-direction: column;
```

Comment:
you can use something like this to replicate what was done in https://github.com/grafana/grafana/pull/52078/files#diff-4d6a1812f83d2610a5195b99f6fd09c162ea4ddb51f7112d781435a0e55a140eL14
```css
      flex-direction: column;
      & + & {
        border-left: 1px solid some color from the theme;
      }
```